ci-jobs/templates/xcuitest-e2e-template.yml

```diff
@@ -38,6 +38,7 @@ jobs:
       xcodeVersion: ${{ parameters.xcodeVersion }}
       deviceName: ${{ parameters.deviceName }}
       vmImage: ${{ parameters.vmImage }}
```

Comment:
>  Is it possible to update the XCode version in CI?

Yes, here can use `macos-12` and set the Xcode version as 14.x
https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md

Perhaps we also could update the entire env to macos-12 and Xcode 12 if possible as another PR. `macos-latest` will be macos-12 base early this year, thus I guess we also could use the instance for non-macOS/Xcode specific environment

Reply:
Thank you! I'll prepare the PR.